index.html

```diff
@@ -231,5 +231,31 @@ <h3 class="section-title contact-title">CONTACT</h3>
         </form>
       </div>
     </section>
+
+    <!-- Footer part -->
+    <footer>
+      <div class="foot-line"></div>
+      <div class="footer-content">        
+        <ul class="social s-foot">
+          <li>
+            <a href=""><i class="fa-brands fa-github-square"></i></a>
```

Comment:
Kindly add your GitHub link after "href" property, it helps to show your code more real.